packages/sparkling-controls/src/calendar/MonthCalendar.svelte

```diff
@@ -0,0 +1,116 @@
+// limitations under the License.
+-->
+
+<script type="ts">
+  import Button from '@anticrm/sparkling-controls/src/Button.svelte'
+  import {
+    areDatesEqual, 
+    day,
+    firstDay,
+    getMonthName,
+    getWeekDayName,
+    incrementMonth,
+    isWeekend,
+    weekday
+  } from './internal/DateUtils';
+
+  export let mondayStart: boolean = false
+  export let weekFormat: "narrow" | "short" | "long" | undefined = 'short'
+  export let selectedDate: Date | undefined = undefined
+
+  let currentDate: Date = new Date()
+  let firstDayOfCurrentMonth: Date = firstDay(currentDate, mondayStart)
+
+  function onSelect(date: Date) {
+    selectedDate = date;
+  }
+
+  function incMonth(count: number) {
+    if (count) {
+      currentDate = incrementMonth(currentDate, count);
+      firstDayOfCurrentMonth = firstDay(currentDate, mondayStart);
+    }
+  }
+</script>
+
+<style lang="scss">
+  .day-name,
+  .selected-month-controller {
+    display: flex;
+    justify-content: center;
+  }
+  .days-of-week-header,
+  .days-of-month {
+    display: grid;
+    grid-template-columns: repeat(7, 1fr);
+  }
+  .weekend {
+    background-color: var(--theme-bg-accent-color);
+  }
+  .today {
+    color: #a66600;
+  }
+  .selected {
+    border-radius: 3px;
+    color: var(--theme-content-dark-color);
+  	background-color: var(--theme-bg-dark-color);
+  }
+  .cell {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .cell:hover {
+    border-radius: 3px;
+    background-color: var(--theme-bg-dark-hover);
+    color: var(--theme-content-color);
+  }
+  .month-name {
+    color: var(--theme-content-dark-color);
+    font-size: 14px;
+    font-weight: bold;
+    margin: 0 5px;
+  }
+</style>
+
+<div class="month-calendar">
+  <div class="selected-month-controller">
+    <Button size="small" on:click={() => incMonth(-1)}>&lt;</Button>
+    <div class="month-name">{getMonthName(currentDate) + " " + currentDate.getFullYear()}</div>
+    <Button size="small" on:click={() => incMonth(1)}>&gt;</Button>
+  </div>
+  <div class="days-of-week-header">
+    {#each [...Array(7).keys()] as dayOfWeek}
+      <div class="day-name">{getWeekDayName(day(firstDayOfCurrentMonth, dayOfWeek), weekFormat)}</div>
+    {/each}
+  </div>
+  <div class="days-of-month">
+    {#each [...Array(6).keys()] as weekIndex}
+      {#each [...Array(7).keys()] as dayOfWeek}
+        <div 
+          class="cell"
+          class:weekend={isWeekend(weekday(firstDayOfCurrentMonth, weekIndex, dayOfWeek))}
+          class:today={areDatesEqual(new Date(), weekday(firstDayOfCurrentMonth, weekIndex, dayOfWeek))}
+          class:selected={areDatesEqual(selectedDate, weekday(firstDayOfCurrentMonth, weekIndex, dayOfWeek))}
+          on:click={() => onSelect(weekday(firstDayOfCurrentMonth, weekIndex, dayOfWeek))}
+        >
+          {weekday(firstDayOfCurrentMonth, weekIndex, dayOfWeek).getDate()}
```

Comment:
Should we try ading a <slot> here to provider a way to customise individual calendar cells?